tools/toolchain/scripts/stage8/install_sirius.sh

```diff
@@ -6,7 +6,7 @@
 [ "${BASH_SOURCE[0]}" ] && SCRIPT_NAME="${BASH_SOURCE[0]}" || SCRIPT_NAME=$0
 SCRIPT_DIR="$(cd "$(dirname "$SCRIPT_NAME")/.." && pwd -P)"
 
-sirius_ver="7.5.2"
+sirius_ver="7.6.0"
 sirius_sha256="9ae01935578532c84f1d0d673dbbcdd490e26be22efa6c4acf7129f9dc1a0c60"
```

Comment:
The new sha256 is: `e424206fecb35bb2082b5c87f0865a9536040e984b88b041e6f7d531f8a65b20`.